frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -44,7 +44,7 @@ export const BuyReceiveCTA = ({ code, unit, balanceList, exchangeBuySupported =
   const isMobile = useMediaQuery('(max-width: 768px)');
 
   const onBuyCTA = () => route(code ? `/buy/info/${code}` : '/buy/info');
-  const onWalletConnect = () => route(`/account/${code}/wallet-connect`);
+  const onWalletConnect = () => route(`/account/${code}/wallet-connect/connect`);
```

Comment:
This will be changed in the future to the "all connected acounts/sessions" screen (according to the mockup) once that screen/page has been created.